distribution/docs/src/main/jdocs/content/_plugins/workspace-pre-ingest-plugin-contents.adoc

```diff
@@ -0,0 +1,22 @@
+:type: plugin
+:status: published
+:title: Workspace Pre-Ingest Plugin
+:link: _workspace_pre_ingest_plugin
+:plugintypes: preingest
+:summary: Verifies that a workspace has an associated email to enable sharing.
+
+The Workspace Pre-Ingest Plugin verifies that a workspace has an associated email to enable sharing and assigns that email as "owner".
```

Comment:
❓ I'm reading this and wondering if it'd be useful to have a (optional?) section to describe what happens when a plugin is disabled. For instance, what would be the effect on Workspaces if this plugin was _not_ installed? This section could also highlight the plugins that are either required or have potentially negative side-effects if disabled. Thoughts?